src/content/app/genome-browser/state/track-settings/trackSettingsSelectors.ts

```diff
@@ -56,3 +58,21 @@ export const getAllTrackSettingsForGenome = (
 ): TrackSettingsForGenome | null => {
   return state.browser.trackSettings[genomeId] ?? null;
 };
+
+export const getAllNonFocusTrackSettingsForGenome = createSelector(
+  getAllTrackSettingsForGenome,
+  (trackSettingsForGenome) => {
+    if (!trackSettingsForGenome) {
+      return null;
+    }
+    const settings: Record<string, TrackSettings> = {};
+    for (const [trackId, trackSettings] of Object.entries(
+      trackSettingsForGenome.settingsForIndividualTracks
+    )) {
+      if (trackId !== 'focus' && trackId !== 'focus-variant') {
```

Comment:
can we do a match for `focus` to find all non-focus tracks?

Reply:
you mean running `/focus/.test(trackId)`?

🤷‍♂️ Do we need to? The risk with the above is that other tracks (which we don't control) will need to be guaranteed to not have a string `f-o-c-u-s` anywhere in their ids.

Reply:
I mean to use regex to match trackId. But yes its not good if there is another track that has matches `focus` but where did `focus-gene` go?

Reply:
`focus-gene` is `focus` 😄